### PR TITLE
simplify dune files for tests

### PR DESCRIPTION
**Pull request by samoht**

### CODE REVIEW

File: test/irmin-pack/dune [hunk 1]
```diff
@@ -1,24 +1,7 @@
...
+(test
  (name test)
- (modules test test_utils)
- (libraries alcotest astring irmin irmin-test test_pack))
-
-(rule
- (alias runtest)
  (package irmin-pack)
- (action
-  (run ./test.exe -q --color=always)))
```
Comment (CraigFe) [inline test/irmin-pack/dune:19]: From what @zshipko describes in https://github.com/mirage/irmin/pull/1313#issuecomment-818840782, the slow tests for `irmin-pack` take many minutes to run (which is fine for CI). What's the new time for a fresh `dune runtest` with these changes?

Comment (samoht) [reply test/irmin-pack/dune:19]: Right, I'm super confused, for some reason I though that `-q` was the default. Would be nice if we could set this up in the test files - but in any case I'm happy to close the PR as this does indeed seems to be a regression.

Comment (samoht) [reply test/irmin-pack/dune:19]: (and that we were doing this for `--color=always` which is now fixed in alcotest)

Comment (zshipko) [reply test/irmin-pack/dune:19]: @CraigFe I actually disabled the longer running test of the two: https://github.com/mirage/irmin/blob/94ccbd2d64eb32f515f577ae58948fa06b76038b/src/irmin-test/store.ml#L2048

**Pull request closed without merging**